mods/ra/rules/player.yaml

```diff
@@ -18,6 +18,7 @@ Player:
 		LowPowerSlowdown: 3
 		QueuedAudio: Building
 		SpeedUp: True
+		BuildTimeSpeedReduction: 100, 75, 60, 50
```

Comment:
The WF is very unique in that its production is not taken full advantage of because of its pricing. Deviating from the rule for this particular queue I believe wouldn't be experienced as confusing and the upsides are imo enough to justify it.